README.md

```
@@ -1,13 +1,14 @@
 ![badge](https://img.shields.io/badge/thank%20you-for%20visiting-brightgreen)
 <br>
-[![Rails](https://img.shields.io/badge/Rails-v7.0.3-%23a72332)](https://rubygems.org/gems/rails/versions/7.0.3)
+[![Ruby](https://img.shields.io/badge/Ruby-v3.4.9-%23CC342D)](https://www.ruby-lang.org/)
+[![Rails](https://img.shields.io/badge/Rails-v8.1-%23a72332)](https://rubygems.org/gems/rails/versions/8.1.0)
 
 # 抹茶と神社。
 
 <img src="https://user-images.githubusercontent.com/94298144/189052247-e5390c19-8701-49c7-83ec-3e9793c8d363.png" width="400" height="400">
```

markdownlint-cli2 (0.23.2)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)